### PR TITLE
Populate Front Page container

### LINKED ISSUE #1768: Populate Homepage container
~~Blocked on https://github.com/bbc/simorgh/pull/1703~~

**Is your feature request related to a problem? Please describe.**
Following https://github.com/bbc/simorgh/pull/1729 https://github.com/bbc/simorgh/pull/1703 we now have a homepage container, and a section container, lets introduce them to eachother 

**Describe the solution you'd like**
Loop over all sections in `frontPageData`, rendering the section container for each one in order.

We may want to consider doing this looping in a seperate container or file due to an aim to keep FrontpageMain clean and tidy, see how it looks when implementing and go with your gut.

To place the content in the center of the page, use a single [GridItemConstrainedLarge](https://github.com/bbc/simorgh/blob/latest/src/app/lib/styledGrid.js#L24-L26) around all sections, this will sort out positioning and the padding. We will probably change this later, but keep it simple for now :)

**Testing notes**
[Tester to complete]

Dev insight: Storybook and unit testing!

### PR DESCRIPTION
**Pull request by DenisHdz**
Resolves #1768

**Overall change:**
Populate the Front Page container using the sections container.

Screenshot 2019-05-24 at 08 48 10

**Code changes:**
- Get `content` from `frontPageData`
- Loop over all sections in `content`, rendering the `FrontPageSection` for each one in order.
- Add  `<GridItemConstrainedLarge>` around all sections to place the content in the center of the page.

---
- [X] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Comment by DenisBBC** [comment]
Please note, that the `StoryPromos` don't have a link and the latest ones are displayed althought they don't have a headline because [this](https://github.com/bbc/simorgh/pull/1769) PR hasn't been merged yet.

File: src/app/containers/FrontPageMain/index.jsx [hunk 1]
```diff
@@ -2,19 +2,21 @@ import React, { Fragment } from 'react';
...
-  );
+  const content = deepGet(['content', 'groups'], frontPageData);
 
   return (
     <Fragment>
       <main role="main">
         <GhostWrapper>
-          <h1>{title}</h1>
+          <GridItemConstrainedLarge>
+            {content.map(group => (
```
Comment (dr3) [inline src/app/containers/FrontPageMain/index.jsx]: Not checked yet, but this likely needs a key

Comment (dr3) [reply src/app/containers/FrontPageMain/index.jsx]: Yeah, bunch of warnings in storybook, can you try and resolve them please?

Comment (DenisHdz) [reply src/app/containers/FrontPageMain/index.jsx]: Most of them are related to [this](https://github.com/bbc/simorgh/issues/1776)

**Review by dr3 — COMMENTED** [review]
Storybook shows the brand as 'news' so I imagine were not setting the context correctly, could we set that correctly please?

Would be nice to have a story per service too :)

**Review by dr3 — COMMENTED** [review]
Can we also pass the fixture data [here](https://github.com/bbc/simorgh/blob/b7004b791d3cb70b0a5b1150086f891a96f7d856/src/app/containers/FrontPage/index.stories.jsx#L3) through the preprocessor? Using the rules in the order of `filter types`, `empty groups`, `top stories merge` please :) (if 1 isnt merged yet, leave it out for now :) )

**Comment by DenisHdz** [comment]
> Storybook shows the brand as 'news' so I imagine were not setting the context correctly, could we set that correctly please?
> 
> Would be nice to have a story per service too :)

I added a story per service but in order to make it work I had to pull some config data from `news` into the `igbo`, `yoruba` and `pidgin` config files as they are incomplete at the [moment](https://github.com/bbc/simorgh/pull/1729/files/bd99160f5bf7973b0c31baec7f63f188b4aa8aed#r285648226).

**Review by dr3 — DISMISSED** [review]
LGTM, can you please write an issue to add some assertion tests for FrontpageMain please? Would be good to get some more coverage

**Review by dr3 — APPROVED** [review]

**Review by j-pendlebury — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
LGTM.

**Pull request merged**